.github/workflows/monthly-pre-release.yml

```diff
@@ -0,0 +1,16 @@
+name: 'Monthly pre-release'
+on: ['push']
```

Comment:
We could actually schedule it if we want this to happen once per month entirely hands-off:
https://docs.github.com/en/actions/reference/events-that-trigger-workflows#schedule